auth/basics/android.md

```diff
@@ -123,8 +171,48 @@ skygear.whoami(new AuthResponseHandler() {
     }
 
     @Override
-    public void onAuthFail(String reason) {
-        // Error handling...
+    public void onAuthFail(Error error) {
+        Log.i("Skygear User", "onAuthFail: Fail with reason:" + error.getMessage());
     }
 });
 ```
+
+## Changing username/email/password
+
+### Change the username / email of a user
+
+To change a user's username and email, you can use the `skygear.saveUser()` method by providing the user ID and the new username and/or the new email.
+
+Every user can only edit his/her information
+While only users with admin privilege can change information of other users.
```

Comment:
The word "admin privilege" will be confusing because this is not mentioned/explained in other places in this document